specification/security/resource-manager/Microsoft.Security/preview/2022-03-01-preview/vmScanners.json

```diff
@@ -0,0 +1,259 @@
+          },
+          "204": {
+            "description": "No Content - Resource does not exist"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Security/vmScanners": {
+      "get": {
+        "description": "Gets the subscription's VM scanner configurations.",
+        "operationId": "VmScanners_ListBySubscription",
+        "x-ms-examples": {
+          "Gets the subscription's VM scanner configurations": {
+            "$ref": "./examples/VmScanners/ListBySubscriptionVmScanners_example.json"
+          }
+        },
+        "tags": [
+          "vmScanners"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ScannerConfigurationList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ScannerConfigurationList": {
+      "type": "object",
+      "description": "Represents a list of VM scanners configurations.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ScannerConfiguration"
+          }
+        }
+      }
+    },
+    "ScannerConfiguration": {
+      "type": "object",
+      "description": "Represents the configuration of the VM scanner",
+      "required": [
+        "properties"
+      ],
+      "properties": {
+        "properties": {
+          "description": "Properties of the VM Scanner resource",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ScannerProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/Resource"
+        }
+      ]
+    },
+    "ScannerProperties": {
+      "type": "object",
+      "description": "VM Scanner resource properties",
+      "properties": {
+        "scanningMode": {
+          "type": "string",
+          "description": "The scanning mode.",
+          "enum": [
+            "Saas"
+          ],
+          "x-ms-enum": {
+            "name": "ScanningMode",
+            "modelAsString": true
+          }
+        },
+        "exclusionTag": {
```

Comment:
Please be more descriptive here. this docs eventually translates to Azure Rest API specs.
Suggestion:

The tag used to exclude resources from being scanned. Machines tagged with that value by this value will not be scanned by this VM scanner configuration.